core/extensions/COMServerDemo/ReadMe.md

```diff
@@ -42,3 +42,5 @@ Program should output an estimated value of &#960;.
 Program should output an estimated value of &#960;.
 
 **Note** The RegFree COM scenario requires a customized [application manifest](https://docs.microsoft.com/windows/desktop/sbscs/manifests) in the executing binary. This means that attempting to execute through `dotnet.exe` will not work and instead trigger a rebuild of the project.
+
+**Note** Running the "Registered COM" first and then immediately follow it by "RegFree COM" will not work, since the build system will not correctly rebuild all files (the simple property change is not detected as a reason for a full rebuild). To fix this, run `dotnet clean` between the two samples.
```

Comment:
```suggestion
**Note** Running the "Registered COM" first and then immediately following it by "RegFree COM" will not work, since the build system will not correctly rebuild all files (the simple property change is not detected as a reason for a full rebuild). To fix this, run `dotnet clean` between the two samples.
```